### PR TITLE
Updating to a simple redirect to the main topic.

### PR DESCRIPTION
**Pull request by stevewhims**
Remove the stub topic, and use the TOC to send the reader to the detailed topic.

### CODE REVIEW

File: docs/core/porting/upgrade-assistant-uwp-framework.md [hunk 1]
```diff
@@ -1,27 +1,12 @@
 ---
-title: Upgrade UWP apps to .NET 6
-description: Use the .NET Upgrade Assistant to migrate a C# UWP app to a Windows UI Library (WinUI) 3 app that uses the Windows App SDK. The .NET Upgrade Assistant is a CLI tool.
+title: Migrate UWP to Windows App SDK with the .NET Upgrade Assistant
+description: The .NET Upgrade Assistant is a Visual Studio extension (preferred), and a command-line tool, that's designed to assist with upgrading apps to the latest version of .NET.
 author: adegeo
-ms.date: 09/20/2022
+ms.date: 09/02/2023
```
Comment (gewarren) [inline docs/core/porting/upgrade-assistant-uwp-framework.md:5]:
```suggestion
ms.date: 06/02/2023
```

File: docs/core/porting/upgrade-assistant-uwp-framework.md [hunk 2]
```diff
@@ -1,27 +1,12 @@
 ---
-title: Upgrade UWP apps to .NET 6
-description: Use the .NET Upgrade Assistant to migrate a C# UWP app to a Windows UI Library (WinUI) 3 app that uses the Windows App SDK. The .NET Upgrade Assistant is a CLI tool.
+title: Migrate UWP to Windows App SDK with the .NET Upgrade Assistant
```
Comment (gewarren) [inline docs/core/porting/upgrade-assistant-uwp-framework.md:2]:
```suggestion
title: Migrate from UWP to Windows App SDK with the .NET Upgrade Assistant
```

**Review by gewarren — APPROVED** [review]

**Comment by stevewhims** [comment]
#sign-off

**Comment by adegeo** [comment]
👀

**Review by adegeo — CHANGES_REQUESTED** [review]
@stevewhims why not just put `/windows/apps/windows-app-sdk/migrate-to-windows-app-sdk/upgrade-assistant` in the TOC instead of this file and then create a redirect?

The toc file is in `/docs/navigate/migration-guide`

**Comment by stevewhims** [comment]
> @stevewhims why not just put `/windows/apps/windows-app-sdk/migrate-to-windows-app-sdk/upgrade-assistant` in the TOC instead of this file and then create a redirect?
> 
> The toc file is in `/docs/navigate/migration-guide`

I've gone with that direction. Does it look ok?

**Comment by adegeo** [comment]
@stevewhims Looks good. I think the redirect should be in `.openpublishing.redirection.core.json`, and the redirect you filed is missing .md at the end of the file name. Once that's fixed, it's merge time! Thanks!

**Comment by stevewhims** [comment]
> @stevewhims Looks good. I think the redirect should be in `.openpublishing.redirection.core.json`, and the redirect you filed is missing .md at the end of the file name. Once that's fixed, it's merge time! Thanks!

Let's try this. Btw, I can't merge AFAIK, so if I eventually get these changes right, it'd be great if you could merge for me. Thank you!

**Review by adegeo — APPROVED** [review]
Sorry for the delay!

**Pull request merged**